test/eradius_test_handler.erl

```diff
@@ -43,7 +43,8 @@ radius_request(#radius_request{cmd = request, attrs = Attrs} = _Req, _Nasprop, _
 	    end;
 	#{?User_Name := <<"Vendor-Dicts">>} ->
 	    case AttrsMap of
-		#{?IM_SSID := _, ?IM_Wifi_Connect_Type := _, ?IM_LI_Location := _} ->
+		#{?IM_SSID := _, ?IM_Wifi_Connect_Type := _, ?IM_LI_Location := Loc} ->
+		    ct:pal("--Location : ~p--", [Loc]),
```

Comment:
is that needed?

Reply:
I tend to overdo the debugging :-/ We're only checking whether it's present, although at some point we should check that it's in fact what we're sending (just in case, and not today).
So yes, it can be removed.